frontend/src/plugin/registry.tsx

```diff
@@ -560,4 +597,20 @@ export function registerGetTokenFunction(override: (cluster: string) => string |
   store.dispatch(setFunctionsToOverride({ getToken: override }));
 }
 
+/**
+ * Add a callback for headlamp events.
+ * @param callback - The callback to add.
+ *
+ * @example
+ *
+ * ```ts
+ * registerHeadlampEventCallback((event) => {
+ *   console.log('Last Headlamp event:', event)
```

Comment:
In the example, can you please show the type and data fields being used?
It would help to include the import here for DefaultHeadlampEvents as well.

Reply:
Will do.